### PR TITLE
Multi tenancy

### PR DESCRIPTION
**Pull request by jonatasrancan**

### CODE REVIEW

File: lib/tasks/db.rake [hunk 1]
```diff
@@ -0,0 +1,8 @@
+namespace :db do
+  task create_tenants: :environment do
+    Apartment.tenant_names.each do |tenant_name|
+      Apartment::Tenant.create(tenant_name)
```
Comment (andreibondarev) [inline lib/tasks/db.rake]: Is there a `find_or_create`?

File: .env.example [hunk 2]
```diff
@@ -1,5 +1,5 @@
 SECRET_KEY_BASE=
-ALLOWED_CORS_URL=
+ALLOWED_CORS_URLS=
```
Comment (andreibondarev) [inline .env.example]: Need to change this in `.circleci/config.yml` as well.

**Comment by jonatasrancan** [comment]
@nakedsushi can you take a look on this?

File: app/jobs/audio_processing.rb [hunk 3]
```diff
@@ -4,6 +4,7 @@ class AudioProcessing < ActiveJob::Base
   @queue = :medium
 
   def perform(opts)
+    Apartment::Tenant.switch!(opts[:tenant])
```
Comment (nakedsushi) [inline app/jobs/audio_processing.rb]: Why use `switch!` instead of `switch` and wrap everything below in a block?

Comment (jonatasrancan) [reply app/jobs/audio_processing.rb]: No specific reason, we can change to that approach.

File: README.md [hunk 4]
```diff
@@ -1,7 +1,18 @@
 # Resound API [![CircleCI](https://circleci.com/gh/ProjectResound/resound-api.svg?style=svg)](https://circleci.com/gh/ProjectResound/resound-api)
-
 The backend that all of Resound's apps hook into. More info about the suite of apps [in the wiki](https://github.com/ProjectResound/planning/wiki)
 
+## IMPORTANT NOTE!!
+This is not a working branch. It's one example of how multi-tenancy would work using the [apartment gem](https://github.com/influitive/apartment).
```
Comment (andreibondarev) [inline README.md]: Probably don't need this comment anymore?

**Comment by andreibondarev** [comment]
Looks good! @nakedsushi We've added different S3 buckets per tenant config.

**Comment by nakedsushi** [comment]
This has already been merged in manually.

**Comment by nakedsushi** [comment]
Sorry, closed the wrong PR when cleaning up.

**Comment by nakedsushi** [comment]
Merged manually: https://github.com/ProjectResound/resound-api/commit/19e270f10657e57091a5575982f49c16ffc5ea58

**Pull request closed without merging**